integration_tests/cli/auth-demo-hex.sh

```diff
@@ -0,0 +1,255 @@
+result=$(echo $passphrase | dcld tx auth approve-add-account --address="$user_address" --info="Alice is approving this account" --from alice --yes)
+check_response "$result" "\"code\": 0"
+
+test_divider
+
+echo "Get $user account and confirm that alice and jack are set as approvers"
+result=$(dcld query auth account --address=$user_address)
+check_response_and_report "$result" $user_address "json"
+check_response_and_report "$result"  $jack_address "json"
+check_response_and_report "$result"  $alice_address "json"
+check_response_and_report "$result"  '"info": "Alice is approving this account"' "json"
+check_response_and_report "$result"  '"info": "Jack is proposing this account"' "json"
+
+test_divider
+
+echo "Get $user account"
+result=$(dcld query auth account --address=$user_address)
+check_response_and_report "$result" "\"address\": \"$user_address\""
+
+test_divider
+
+echo "Get a proposed account for $user is not found"
+result=$(dcld query auth proposed-account --address=$user_address)
+check_response "$result" "Not Found"
+
+test_divider
+
+productName="Device #1"
+echo "$user adds Model with VID: $vid_in_hex_format PID: $pid"
+result=$(echo "test1234" | dcld tx model add-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format --productName="$productName" --productLabel="Device Description"   --commissioningCustomFlow=0 --deviceTypeID=12 --partNumber=12 --from=$user_address --yes)
+check_response_and_report "$result" "\"code\": 0"
+
+test_divider
+
+vid_plus_one_in_hex_format=0xA14
+vidPlusOne=$((vid_in_hex_format+1))
+echo "$user adds Model with a VID: $vid_plus_one_in_hex_format PID: $pid_in_hex_format, This fails with Permission denied as the VID is not associated with this vendor account."
+result=$(echo "test1234" | dcld tx model add-model --vid=$vid_plus_one_in_hex_format --pid=$pid_in_hex_format --productName="$productName" --productLabel="Device Description"   --commissioningCustomFlow=0 --deviceTypeID=12 --partNumber=12 --from=$user_address --yes 2>&1) || true
+check_response_and_report "$result" "transaction should be signed by a vendor account containing the vendorID $vidPlusOne"
+
+test_divider
+
+echo "$user updates Model with VID: $vid_in_hex_format PID: $pid_in_hex_format"
+result=$(echo "test1234" | dcld tx model update-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format --productName="$productName" --productLabel="Device Description" --partNumber=12 --from=$user_address --yes)
+check_response_and_report "$result" "\"code\": 0"
+
+test_divider
+
+echo "Get Model with VID: $vid_in_hex_format PID: $pid_in_hex_format"
+result=$(dcld query model get-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format)
+check_response "$result" "\"vid\": $vid"
+check_response "$result" "\"pid\": $pid"
+check_response "$result" "\"productName\": \"$productName\""
+
+test_divider
+
+echo "Get $user account and confirm that alice and jack are set still present as approvers, also the sequence number should be 3 as we executed 3 txs"
```

Comment:
I believe all below lines are not related to this test. Account revocation has been already tested by other tests.

Reply:
Done! ✅